mkdocs.yml

```diff
@@ -7,39 +7,113 @@ nav:
+      - Options: configuration/options.md
+      - Search: configuration/search.md
+      - Memory: configuration/memory.md
+      - Voice: configuration/voice.md
+      - Image Generation: configuration/imagegen.md
 
   - Help us improve Auto-GPT:
-    - Share your debug logs with us: share-your-logs.md
-    - Contribution guide: contributing.md
-    - Running tests: testing.md
-    - Code of Conduct: code-of-conduct.md
+      - Share your debug logs with us: share-your-logs.md
+      - Contribution guide: contributing.md
+      - Running tests: testing.md
+      - Code of Conduct: code-of-conduct.md
 
   - Challenges:
-    - Introduction: challenges/introduction.md
-    - List of Challenges:
-      - Memory:
-        - Introduction: challenges/memory/introduction.md
-        - Memory Challenge A: challenges/memory/challenge_a.md
-        - Memory Challenge B: challenges/memory/challenge_b.md
-        - Memory Challenge C: challenges/memory/challenge_c.md
-        - Memory Challenge D: challenges/memory/challenge_d.md
-      - Information retrieval:
-        - Introduction: challenges/information_retrieval/introduction.md
-        - Information Retrieval Challenge A: challenges/information_retrieval/challenge_a.md
-        - Information Retrieval Challenge B: challenges/information_retrieval/challenge_b.md
+      - Introduction: challenges/introduction.md
+      - List of Challenges:
+          - Memory:
+              - Introduction: challenges/memory/introduction.md
+              - Memory Challenge A: challenges/memory/challenge_a.md
+              - Memory Challenge B: challenges/memory/challenge_b.md
+              - Memory Challenge C: challenges/memory/challenge_c.md
+              - Memory Challenge D: challenges/memory/challenge_d.md
+          - Information retrieval:
+              - Introduction: challenges/information_retrieval/introduction.md
+              - Information Retrieval Challenge A: challenges/information_retrieval/challenge_a.md
+              - Information Retrieval Challenge B: challenges/information_retrieval/challenge_b.md
   - Submit a Challenge: challenges/submit.md
   - Beat a Challenge: challenges/beat.md
 
   - License: https://github.com/Significant-Gravitas/Auto-GPT/blob/master/LICENSE
 
-theme: readthedocs
+theme:
+  name: material
+  icon:
+    logo: material/book-open-variant
+  favicon: imgs/Auto_GPT_Logo.png
+  features:
+    - navigation.sections
+    - toc.follow
+    - navigation.top
+    - content.code.copy
+  palette:
+    # Palette toggle for light mode
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      toggle:
+        icon: material/weather-night
+        name: Switch to dark mode
+
+    # Palette toggle for dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      toggle:
+        icon: material/weather-sunny
+        name: Switch to light mode
 
 markdown_extensions:
-  admonition:
-  codehilite:
```

Comment:
This is used in indented code blocks and should not be removed without updating those blocks

Reply:
Good catch. I've updated the codeblocks.

codehilite (extension) is superceded by Highlight, but requires changes to the code blocks.
Ref: https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#highlight